### ISSUE TITLE
LoginManager.token_loader removed from flask-login.

### ISSUE THREAD

**Issue by elektito** [lingthio/Flask-User#141]
`LoginManager.token_loader` has been removed in flask_login in commit f4ff19c, but it is used in flask_user, causing an error like this in my app:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/mostafa/source/dt/app.py", line 13, in <module>
    user_manager = UserManager(db_adapter, app)
  File "/home/mostafa/source/dt/venv/lib/python3.5/site-packages/flask_user/__init__.py", line 53, in __init__
    self.init_app(app, db_adapter, **kwargs)
  File "/home/mostafa/source/dt/venv/lib/python3.5/site-packages/flask_user/__init__.py", line 166, in init_app
    self.setup_login_manager(app)
  File "/home/mostafa/source/dt/venv/lib/python3.5/site-packages/flask_user/__init__.py", line 199, in setup_login_manager
    @self.login_manager.token_loader
AttributeError: 'LoginManager' object has no attribute 'token_loader'

```

**Comment by tarnacious** [comment]
I just removed the token_loader and it seems to work ok.

**Comment by ramidas** [comment]
From CHANGES:

Version 0.4.0
-------------

Released on October 26th, 2016

- Fixes OPTIONS exemption from login. #244
- Fixes use of MD5 by replacing with SHA512. #264
- BREAKING: The `login_manager.token_handler` function, `get_auth_token` method
  on the User class, and the `utils.make_secure_token` utility function have
  been removed to prevent users from creating insecure auth implementations.
  Use the `Alternative Tokens` example from the docs instead. #291

**Comment by lingthio** [comment]
Released in v0.6.9.
